CONTRIBUTING.md

```diff
@@ -110,40 +107,39 @@ you might find out that you don't need to create one.
 
 When you are creating an enhancement suggestion, please 
 [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). 
-Fill in the [issue_template.md](.github/issue_template.md), including the steps that you imagine you 
-would take if the feature you're requesting existed.
 
 #### Before Submitting An Enhancement Suggestion
 
 * **Check the [Debugging documentation].** You might be able to find the cause of the problem and fix things yourself. 
-* **Perform a [cursory search of project issues](https://github.com/search?q=+is%3Aissue+repo%3APegasysEng/Pantheon)** 
+* **Perform a [cursory search of project issues](https://pegasys1.atlassian.net/secure/Dashboard.jspa)** 
 to see if the problem has already been reported. If it has **and the issue is still open**, add a comment 
 to the existing issue instead of opening a new one.
 
 #### How Do I Submit A (Good) Enhancement Suggestion?
 
-Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).
-Issues should provide the following information by filling in the [issue_template.md](.github/issue_template.md) 
-and providing the following information:
+Enhancement suggestions are tracked as [Jira issues](https://pegasys1.atlassian.net/secure/Dashboard.jspa).
+Provide the following information:
 
 * **Use a clear and descriptive title** for the issue to identify the suggestion.
 * **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
-* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use 
-in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Provide specific examples to demonstrate the steps**. If you're providing code snippets in the issue, 
+ use backticks (````) to format the code snippets..
 * **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
-* **Include screenshots** which help you demonstrate the steps.
+* **Include screenshots** which help you demonstrate the steps where possible.
 * **Explain why this enhancement would be useful** to most users.
 * **Does this enhancement exist in other clients?**
 * **Specify which version of the software you're using.** You can get the exact version by running 
 `pantheon -v` in your terminal.
 * **Specify the name and version of the OS you're using.**
 
 ## Your First Contribution
-Start by looking through the 'good first issue' and 'help wanted' labeled issues:
+Start by looking through the 'good first issue' and 'help wanted' labeled issues on the [Jira dashboard](https://pegasys1.atlassian.net/secure/Dashboard.jspa):
```

Comment:
This will always load their last looked at dashboard. Do we want to specifically send them to the system dashboard (https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000)?

Reply:
Yes - fixed